src/irmin/s.ml

```diff
@@ -360,10 +360,10 @@ module type NODE_GRAPH = sig
     min:node list ->
     max:node list ->
     ?node:(node -> unit Lwt.t) ->
-    ?contents:(contents * metadata -> unit Lwt.t) ->
+    ?contents:(contents -> unit Lwt.t) ->
     ?edge:(node -> node -> unit Lwt.t) ->
-    ?skip_nodes:(node -> bool Lwt.t) ->
-    ?skip_contents:(contents * metadata -> bool Lwt.t) ->
+    ?skip_node:(node -> bool Lwt.t) ->
```

Comment:
GitHub won't let me suggest it, but need to `s/skip_nodes/skip_node/` in comment below.